helm-chart/images/binderhub/binderhub_config.py

```diff
@@ -29,6 +29,12 @@ def get_config(key, default=None):
 
 c.BinderHub.builder_image_spec = get_config('binder.repo2docker-image')
 c.BinderHub.build_node_selector = get_config('binder.build-node-selector', {})
+appendix = get_config('binder.appendix')
+if appendix:
+    c.BinderHub.appendix = appendix
+else:
+    c.BinderHub.binderhub_appendix_url = get_config('binder.appendix-url', '')
```

Comment:
Why do we only set this `if not appendix`? My first guess would have been that you can set appendix and `appendix_url` independently of each other. A custom appendix might not use `appendix_url` but ok. I think I have a slight preference for letting people set a custom appendix and use the appendix_url config option. But I couldn't justify it with a use-case right now :-/

If we don't change it add a comment saying "We assume that a custom appendix already contains all the right URLs and information".